packages/e2e-test-utils-playwright/src/admin/visit-site-editor.ts

```diff
@@ -66,5 +66,5 @@ export async function visitSiteEditor(
 	// TODO: Ideally the content underneath the spinner should be marked inert until it's ready.
 	await this.page
 		.locator( '.edit-site-canvas-spinner' )
-		.waitFor( { state: 'hidden' } );
+		.waitFor( { state: 'hidden', timeout: 60_000 } );
```

Comment:
This was needed because, for performance tests, we load [an enormous post fixture](https://github.com/WordPress/gutenberg/blob/refactor/playwright-performance-tests-p2/test/performance/assets/large-post.html) to test against, which often takes more than the default 10 seconds to load.

I'm not sure the rationale for using a post big enough to freeze the browser for a bit, but it's probably something to be discussed in a separate PR? /cc @youknowriad

Reply:
when I've tested with the large post in my browser it opens reasonably fast, but if I open the profiling tools I've seen it freeze for more than a minute.

I'm inferring that Chrome is spending a lot of time instrumenting the thousands of listeners and hooks when we turn of tracing, which probably happens here as well.

so this may not be realistic from a user perspective, but the wait may still occur when testing and profiling

Reply:
Yeah, profiling adds a lot of overhead. We could assume that the overhead added shares averagely across all code and can be seen as a fair emulated slowdown of the CPU, but I don't know if that's true 😅 .

Reply:
it's weird. I don't understand it, but it doesn't seem to appear in the metrics. I suspect that the profiling mechanisms don't start until after this initialization. the page isn't interactive while it's starting up, but then once it finishes things are suddenly fast and fine.

so it may not be the case that Chrome reports this big delay, but it still delays the tests. based on some recent profiling work I'm wondering if we might have code paths that are to blame. there are so many listeners involved and it doesn't seem reasonable that we should have so many. maybe Chrome's profiler simply chokes at tens of thousands of listeners (or maybe this is a normal number 🤷‍♂️ I really don't know).